packages/tooltip/src/Tooltip.tsx

```diff
@@ -62,7 +62,7 @@ const baseStyles = css`
   padding: 12px ${borderRadius}px;
   box-shadow: 0px 2px 4px -1px ${transparentize(0.85, palette.black)};
   cursor: default;
-  overflow-wrap: break-word;
+  word-break: break-all;
```

Comment:
This would break every word, even small words. I tested with `thisisoner testingtesitng ok what about now` and `about` was split apart.
![Screen Shot 2022-08-02 at 11 18 46 AM](https://user-images.githubusercontent.com/5740159/182410761-33a044a7-3ebf-40dd-9a27-21bc38fc7206.png)

Reply:
Adding this to the inner div (where `baseTypeStyle` is applied) seems to resolve the issue: 
```
width: 100%;
overflow-wrap: anywhere;
```

Reply:
@shaneeza I thought `break-all` handles this logic of prioritizing words, but I guess I was wrong.

Thank you @TheSonOfThomp! I was deep in SO and struggling to find something that worked - this is CLEAN 🙏🙏🙏

Reply:
Yeah nice save Adam!!!